common/app/views/fragments/amp/stylesheets/tones.scala.html

```diff
@@ -21,7 +21,8 @@
     color: #fff;
 }
 
-.tonal--tone-comment .tonal__standfirst {
+.tonal--tone-comment .tonal__standfirst,
```

Comment:
Hmm. We need clarity on whether that rule is actually enforced or not. Will ask Google, but carry on as you are for now.